common/web/types/src/ldml-keyboard/pattern-parser.ts

```diff
@@ -40,6 +47,25 @@ export class MarkerParser {
    */
   public static readonly ANY_MARKER_ID = '.';
 
+  /**
+   * Marker sentinel as a string - U+FFFF
+   */
+  public static readonly SENTINEL = String.fromCodePoint(constants.marker_sentinel);
+
+  /**
+   * Matches all markers.
+   */
+  public static readonly SENTINEL_ALL_MARKERS = this.SENTINEL + this.SENTINEL;
```

Comment:
This makes me a little squirmy. I'm probably worrying about nothing but it does mean `SENTINEL` has two meanings, whereas in kmx processor `SENTINEL` only has a single meaning.

FWIW, in kmx processor, `SENTINEL` is always followed by a `CODE` to determine the actual meaning of the sentinel, e.g. `SENTINEL CODE_DEADKEY`. I wonder if we should be following the same pattern here to allow for future expansion

Reply:
@mcdurdin i actually couldn't find `U_SENTINEL` or `SENTINEL` in the keyman source…  I did see SENTINEL_CODE_MARKER which is FDD0 but it didn't seem to go with any other sentinel value. 

I did delete SENTINEL_ALL_MARKERS in a future PR

Reply:
Yep, we discussed this at standup but for reference, `UC_SENTINEL CODE_DEADKEY <marker>` is the sequence used by kmx.